ajax/question_delete.php

```diff
@@ -39,6 +39,7 @@
 $questionId = (int) $_REQUEST['id'];
 $question = new PluginFormcreatorQuestion();
 if (!$question->getFromDB($questionId)) {
+    echo __('Question not found', 'formcreator');
```

Comment:
if deletion fails, no error displayed, fixing it

Reply:
The error mesage is processed by js/scripts.js. Your sgestion is possible, but also requires an update in the JS  code. This PR is complicated enough, then we may do this in a separate enhancement PR.